package.json

```diff
@@ -18,6 +18,7 @@
     "test:outdated": "npm outdated --color always --silent || true",
     "test:security": "retire && npm run test:outdated",
     "test:lint": "eslint --ext .js,.json .",
+    "test:docs": "blcl ./docs -ro",
```

Comment:
We don't run this on `npm test`?

Reply:
We can't. Reason is that sometimes you want to link to, say, the API doc, but it hasn't been published yet.

Reply:
Maybe run it in a prepublish hook or something?

Reply:
We could... just not sure if there might be any other caveats I am not thinking of. 

I'd say we could add this to pre-publish at a later time, once we've used that tool a bit more by hand.